src/test/java/com/google/api/codegen/testdata/library_config.baseline

```diff
@@ -16,10 +16,12 @@ language_settings:
 interfaces:
 - name: google.example.library.v1.LibraryService
   collections:
+  - name_pattern: archives/{archive_path=**}/books/{book}
+    entity_name: book
```

Comment:
Why call the special archived one 'book' and the normal shelved one 'book_2'? I would expect the reverse.